library/yum.py

```diff
@@ -105,7 +105,7 @@
 
 
 def is_installed(name):
-    rc, _, _ = module.run_command(['rpmquery', '--quiet', name])
+    rc, _, _ = module.run_command(['rpmquery', '--whatprovides', name])
```

Comment:
It doesn't really matter either way.  I left the quiet off since a simply copy and paste displays information, and seems a little more informative.  Modules do not display output to ansible.